src/main/java/pages/BaseTest.java

```diff
@@ -0,0 +1,24 @@
+package pages;
+
+import org.junit.jupiter.api.AfterEach;
+import org.openqa.selenium.WebDriver;
+
+public class BaseTest {
+    protected static WebDriver driver;
+
+    public BaseTest() {
```

Comment:
remove constructor, just use `@BeforeEach` method here to init driver and do some initial steps such as navigating to the login page etc.